backend/server.py

```
@@ -7874,6 +7874,123 @@ async def legacy_scheduler_toggle(
+    import base64
+    try:
+        from voice.tts import synthesize
+        audio_bytes = await synthesize(body.text)
+        if audio_bytes:
+            return {
+                "audio_b64": base64.b64encode(audio_bytes).decode(),
+                "format": "ogg",
+                "duration_s": round(len(audio_bytes) / 4000, 1),
+            }
+        return {"audio_b64": "", "error": "TTS synthesis returned empty"}
+    except Exception as exc:
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)